### PR TITLE
Add Brainstorm to README

### PR DESCRIPTION
**Pull request by cMadan**

### CODE REVIEW

**Comment by Remi-Gau** [comment]
Hey @cMadan 

Thanks for opening the PR.

Is that the link to the BIDS relevant part: https://neuroimage.usc.edu/brainstorm/ExportBids ?

**Comment by martcous** [comment]
Hello, yes Brainstorm supports both importing and exporting to BIDS (both MEG-BIDS and EEG-BIDS). The link you posted above is the documentation I wrote for exporting BIDS.

**Comment by cMadan** [comment]
I tried a few times over the last few hours, but the above link keeps giving me an error (`neuroimage.usc.edu took too long to respond.`). Regardless, that link is specific to exporting BIDS. 

The link in the PR edit is to Brainstorm as a whole (https://github.com/brainstorm-tools/brainstorm3), which also imports BIDS (as @martcous says). There are also a few example scripts for loading existing BIDS datasets that are also useful, so I thought linking to the toolbox as a whole is the best option.

**Comment by guiomar** [comment]
Thanks Chris, Remi and Martin!

Indeed, Brainstorm can both import and export BIDS datasets, and there are several tutorials using BIDS datasets from OpenNeuro. Thanks for including it!

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Starting point was `spm_BIDS.m` from [SPM12](https://github.com/spm/spm12) ([doc
 ## Other tools (MATLAB only)
 - [dicm2nii](https://github.com/xiangruili/dicm2nii): A DICOM to BIDS converter
 - [imtool3D_BIDS](https://github.com/tanguyduval/imtool3D_td): A 3D viewer for BIDS directory
+- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS import functionality)
```
Comment (Remi-Gau) [inline README.md:55]:
```suggestion
- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS [import](https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega#BIDS_specifications) and [export](https://neuroimage.usc.edu/brainstorm/ExportBids) functionality)
```

Just a suggestion to have the link in there directly to some import and export sections. 

What do you think?

Comment (guiomar) [reply README.md:55]: maybe @martcous you could update the BIDS tutorial to include both import and export? The import part should be quite straight forward.

And the we can update the text like this:

```suggestion
- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS [import and export] (https://neuroimage.usc.edu/brainstorm/ExportBids) and different examples dealing with BIDS datasets (e.g. [group analysis from a MEG visual dataset](https://neuroimage.usc.edu/brainstorm/Tutorials/VisualGroup, [resting state analysis from OMEGA datasets](https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega#BIDS_specifications) )
```

Comment (Remi-Gau) [reply README.md:55]: Oh I had not seen your suggestions @guiomar  and thumb up @cMadan.

Will update and merge now. Sorry for being slow on this.

File: README.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ Starting point was `spm_BIDS.m` from [SPM12](https://github.com/spm/spm12) ([doc
 ## Other tools (MATLAB only)
 - [dicm2nii](https://github.com/xiangruili/dicm2nii): A DICOM to BIDS converter
 - [imtool3D_BIDS](https://github.com/tanguyduval/imtool3D_td): A 3D viewer for BIDS directory
+- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS [import and export] (https://neuroimage.usc.edu/brainstorm/ExportBids) and different examples dealing with BIDS datasets (e.g. [group analysis from a MEG visual dataset](https://neuroimage.usc.edu/brainstorm/Tutorials/VisualGroup, [resting state analysis from OMEGA datasets](https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega#BIDS_specifications) )
```
Comment (Remi-Gau) [inline README.md:55]:
```suggestion
- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS [import and export](https://neuroimage.usc.edu/brainstorm/ExportBids) and different examples dealing with BIDS datasets (e.g. [group analysis from a MEG visual dataset](https://neuroimage.usc.edu/brainstorm/Tutorials/VisualGroup), [resting state analysis from OMEGA datasets](https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega#BIDS_specifications) )
```

**Review by Remi-Gau — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (guiomar) [reply README.md]: Sorry, maybe I placed them in the wrong place. Thanks for taking the time to include them :)

Comment (Remi-Gau) [reply README.md]: no worries

**Comment by Remi-Gau** [comment]
@all-contributors please add @cMadan for content

**Comment by Remi-Gau** [comment]
@all-contributors please add @guiomar for review

**Pull request merged**